pkg/ocm/service_logs.go

```diff
@@ -0,0 +1,9 @@
+package ocm
+
+import (
+	slv1 "github.com/openshift-online/ocm-sdk-go/servicelogs/v1"
+)
+
```

Comment:
@renan-campos I'd also like to see a unit test for this as I'm trying to get us to the point where we're not adding new code that isn't tested. Take a look at `kubeletconfig_test` in the `ocm` package for how we're currently testing interactions with the OCM API.